lib/content-services/src/lib/aspect-list/aspect-list-dialog.component.spec.ts

```diff
@@ -303,6 +303,17 @@ describe('AspectListDialogComponent', () => {
             expect(customAspectCheckbox).not.toBeNull();
             expect(customAspectCheckbox.checked).toBeTruthy();
         });
+
+        it('Should apply button disabled till aspect-list updated', async () => {
+            await fixture.detectChanges();
+            expect(fixture.nativeElement.querySelector('#aspect-list-dialog-actions-apply')).toBeDefined();
+            expect(fixture.nativeElement.querySelector('#aspect-list-dialog-actions-apply').disabled).toBe(true);
+
+            fixture.nativeElement.querySelector('#aspect-list-dialog-actions-clear').click();
+
+            await fixture.detectChanges();
+            expect(fixture.nativeElement.querySelector('#aspect-list-dialog-actions-apply').disabled).toBe(false);
```

Comment:
can we store every element we query into a variable? In this way we can debug them easier in the future